test/accessibility/apps/discover.ts

```diff
@@ -35,6 +36,7 @@ export default function({ getService, getPageObjects }: FtrProviderContext) {
 
   describe('Discover', () => {
     before(async () => {
+      await security.testUser.setRoles(['kibana_admin', 'test_logstash_reader']);
```

Comment:
I think we don't actually need any changes in this file because you're calling setRoles with the default roles.